dear_bindings.py

```diff
@@ -387,6 +363,42 @@ def convert_header(
                                                           'flags',
                                                           'popup_flags'
                                                       ])
+        
+    if is_probably_imgui_internal:
+        # Some functions in imgui_internal already have the Ex suffix,
+        # which wreaks havok on disambiguation
+        mod_rename_functions.apply(main_src_root, {
+            'ImGui_BeginMenuEx': 'ImGui_BeginMenuWithIcon',
+            'ImGui_MenuItemEx': 'ImGui_MenuItemWithIcon',
+            'ImGui_BeginTableEx': 'ImGui_BeginTableWithID',
+            'ImGui_ButtonEx': 'ImGui_ButtonWithFlags',
+            'ImGui_ImageButtonEx': 'ImGui_ImageButtonWithFlags',
+            'ImGui_InputTextEx': 'ImGui_InputTextWithHintAndSize',
+        })
+        
+    mod_disambiguate_functions.apply(dom_root,
```

Comment:
I've did the change as you suggested, however, for this specific example, seems like there's an issue unrelated to this branch: two overloads of `ImGui_PlotLines` are not disambiguated into `ImGui_PlotLinesCallback` and `ImGui_PlotLinesCallbackEx`, but instead, it's `ImGui_PlotLines` and `ImGui_PlotLinesCallback`. Slightly confusing, since they are both callback...